pkg/system/chtimes.go

```diff
@@ -8,8 +8,8 @@ import (
 // Chtimes changes the access time and modified time of a file at the given path
 func Chtimes(name string, atime time.Time, mtime time.Time) error {
 	unixMinTime := time.Unix(0, 0)
-	// The max Unix time is 33 bits set
-	unixMaxTime := unixMinTime.Add((1<<33 - 1) * time.Second)
+	// The max Unix time is 63 bits set
+	unixMaxTime := time.Unix(0, 1<<63-1)
```

Comment:
From the golang time package, "It is valid to pass nsec outside the range [0, 999999999]". The function does the correct conversions to seconds and nanoseconds by design. 

Note: I've got another change to be made here to solve the bug, just trying to set up a machine to repro the bug to test it.